app/main.py

```diff
@@ -1,8 +1,40 @@
 class Car:
-    # write your code here
-    pass
+
+    def __init__(self, comfort_class: int,
```

Comment:
Plase each argument here on a new line including self for better readability, so that opening and closing parentheses `()` will not have arguments on the same line. It would improve readability for other developers.